### PR TITLE
Feat/m2 526 back button

### PR DESCRIPTION
**Pull request by BamMironov**

### CODE REVIEW

File: src/entities/identity/lib/nameUtils.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+export function getFirstName(fullName: string): string {
```
Comment (anq83) [inline src/entities/identity/lib/nameUtils.ts:1]: The function itself looks like workaround, BE should return splitted data

File: src/screens/ui/RootNavigator.tsx [hunk 2]
```diff
@@ -89,21 +97,15 @@ export default () => {
...
         options={{
-          title: 'Applets',
           headerStyle: {
             backgroundColor: colors.lighterGrey2,
           },
           headerTitleStyle: {
             fontSize: 16,
             color: colors.tertiary,
           },
-          headerLeft: () => (
```
Comment (anq83) [inline src/screens/ui/RootNavigator.tsx:100]: Can we leave it temporarilly in order to change user if we don't have logout yet?
Or just to have possibility to go to login screen

**Review by anq83 — APPROVED** [review]

File: src/entities/identity/lib/nameUtils.ts (hunk 1 above)
Comment (BamMironov) [reply src/entities/identity/lib/nameUtils.ts:1]: That's exactly what it is. Yes, the back-end should return first and last names as independent fields

File: src/screens/ui/RootNavigator.tsx (hunk 2 above)
Comment (BamMironov) [reply src/screens/ui/RootNavigator.tsx:100]: We are implementing logout feature today, so no worries

**Pull request merged**